components/docs-chef-io/content/automate/ha_troubleshooting.md

```diff
@@ -53,15 +55,21 @@ The restore command fails when other users or services access the nodes' databas
 ### Issue: Cached Artifact not found in Offline Mode
 
 The cached artifact does not exist in offline mode. This issue occurs in an air gap environment when a package tries to pull any dependency components or details from the internet.
-![Cache Artifact Error](/images/automate/ha_faq_cache.png)
+
+```bash
+"level=error msg=""Failed to restore services"" backup_id=20210913105135 error=""msg=\""failed to install\"""
```

Comment:
These lines don't seem quoted correctly, each line in the final output is being wrapped with double quotes when they should just be the bare line.